index.html

```diff
@@ -34,12 +271,163 @@ <h1>I’m Omayio Glad to see you!</h1>
+          <li><a href="#"><img src="imgs/Github.png" alt="Github Icon" /></a></li>
+          <li><a href="#"><img src="imgs/hands.png" alt="hands" /></a></li>
         </ul><br><br><br><br><br><br><br>
+      </article>
+
+      <section class="grid-container">
+        <div class="grid-item show-case">
+          <div class="content">
+            <h4>Tonic</h4>
+            <ul class="flex">
+              <li class="cano">CANOPY</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="flex ocontent">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button type="submit">See Project</button>
+        </div>
+        <div class="grid-item show-case2">
+          <div class="content">
+            <h4>Multi-Post Stories</h4>
+            <ul class="flex">
+              <li class="cano">CANOPY</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="flex ocontent">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button type="submit">See Project</button>
+        </div>
+        <div class="grid-item show-case3">
+          <div class="content">
+            <h4>Tonic</h4>
+            <ul class="flex">
+              <li class="cano">CANOPY</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="flex ocontent">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button type="submit">See Project</button>
+        </div>
+        <div class="grid-item show-case4">
+          <div class="content">
+            <h4>Multi-Post Stories</h4>
+            <ul class="flex">
+              <li class="cano">CANOPY</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="flex ocontent">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button type="submit">See Project</button>
+        </div>
       </section>
-  </body>
+      <section id="about-me" class="flex">
+        <h2>About Myself</h2>
```

Comment:
- [ ] This section of your page is matched with the Figma design. Kindly work on it. :+1: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![ab1](https://user-images.githubusercontent.com/70483835/166124227-da803e47-5456-40ba-8340-f5b937d370e9.png) | ![ab2](https://user-images.githubusercontent.com/70483835/166124224-f7470cf3-68b3-43be-9cb0-e2a287fa3f09.png)